MC/bin/o2dpg_sim_workflow_anchored.py

```diff
@@ -306,26 +321,47 @@ def main():
     currenttime = GLOparams["SOR"] + prod_offset * GLOparams["OrbitsPerTF"] * LHCOrbitMUS // 1000 # timestamp in milliseconds
     print ("Production put at time : " + str(currenttime))
 
+    # retrieve the GRPHCIF object
+    grplhcif = retrieve_GRPLHCIF(ccdbreader, int(currenttime))
+    eCM = grplhcif.getSqrtS()
+    A1 = grplhcif.getAtomicNumberB1()
+    A2 = grplhcif.getAtomicNumberB2()
+
+    # determine collision system and energy
+    print ("Determined eMC ", eCM)
+    print ("Determined atomic number A1 ", A1)
+    print ("Determined atomic number A2 ", A2)
+    ColSystem = ""
+    if A1 == 82 and A2 == 82:
+      ColSystem = "PbPb"
+    elif A1 == 1 and A2 == 1:
+      ColSystem = "pp"
+    else:
+      print ("Unknown collision system ... exiting")
+      exit (1)
+
+    print ("Collision system ", ColSystem)
```

Comment:
@catalinristea : You no longer need to pass collysion system and energy from the outside

Reply:
Hello @sawenzel , 
Should we already foresee pPb? Even if it is not clear when it will happen in Run 3 (when also OO will take place).